backend/app/src/main/java/com/moh/phlat/backend/esb/json/MaintainFacilityResponse.java

```diff
@@ -0,0 +1,139 @@
+package com.moh.phlat.backend.esb.json;
+
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TimeZone;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.moh.phlat.backend.esb.boundary.PlrToken;
+import com.moh.phlat.backend.model.Control;
+import com.moh.phlat.backend.model.ProcessData;
+
+public class MaintainFacilityResponse implements PlrResponse {
```

Comment:
We need to discuss the design for this. The PLRResponse class should encapsulate these three responses as attributes. You can then implement either a single method or three separate methods to convert the responses from facility, hds, and relation in the process data table, which is currently not being done as a part of phaseA work. Then we can get rid of Interface. we can discuss options.

Reply:
PlrDataLoad is serving this purpose now. As for the interface, I think we discussed possibly leaving this discussion/reworking for another time as part of code cleanup after the load functionality is finalized, and I'd like to go ahead with that plan; I'd rather make sure we get the load functionality done first, given the limit on budget and the timeframe

Reply:
I gave this comment because now you have introduced MaintainResults to hold each result. Then what's the use of PLRResponse. If it is still needed, you can keep it.

Reply:
Ah right, I mixed up PlrDataLoad with MaintainResults. Regardless, yeah there's still things in PLRResponse that are useful to the response objects, so I'll keep it as is for now and then we can revisit this as part of the interface refactoring if/when we get to that after the Stage A work is done